### PR TITLE
Added error description

### LINKED ISSUE #1049: when upload fails because of not enough quota user does not get feedback about the reason
### Steps to reproduce
1. set quota for user
2. upload a file that would exceed the quota

### Expected behaviour
Error saying that the upload failed because of quota exceeded

### Actual behaviour
Error message only says that the upload failed

### PR DESCRIPTION
**Pull request by LukasHirt**
## Description
Upgraded ods and added desc for upload error

## Related Issue

- Fixes #1049 

## How Has This Been Tested?

- test environment: Manually
- test case 1: Set quota for user
- test case 2: Try to upload file bigger than allowed quota

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

## Open tasks:
- [x] Merge https://github.com/owncloud/owncloud-design-system/pull/274 and update ods in Phoenix

### CODE REVIEW

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2739/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2739/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2742/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2742/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2742/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2742/

File: tests/acceptance/features/webUIFiles/createFolders.feature [hunk 1]
```diff
@@ -19,7 +19,7 @@ Feature: create folders
 
   Scenario: Create a folder with existing name
     When the user creates a folder with the invalid name "simple-folder" using the webUI
-    Then the error message 'Creating folder failed…' should be displayed on the webUI
+    Then the error message 'Creating folder failed… The resource you tried to create already exists' should be displayed on the webUI
```
Comment (DeepDiver1975) [inline tests/acceptance/features/webUIFiles/createFolders.feature]: Please consider testing for the title only - the detailed message is server generated and might differ over time, version, environement.

For us it is enough to test if the error message is displayed at this point.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary><p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFolders</strong> failed. Please find the screenshots inside ...</summary><p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIUpload</strong> failed. Please find the screenshots inside ...</summary><p>














</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFiles</strong> failed. Please find the screenshots inside ...</summary><p>


















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>




























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary><p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFolders</strong> failed. Please find the screenshots inside ...</summary><p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIUpload</strong> failed. Please find the screenshots inside ...</summary><p>














</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFiles</strong> failed. Please find the screenshots inside ...</summary><p>


















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>




























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>




</p></details>

**Pull request merged**